### PR TITLE
Updated PR template

### PR DESCRIPTION
**Pull request by pranavgoel29**
## What type of PR is this? (check all applicable)

- [X] ♻️ Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [X] 👷 Optimization
- [X] 📝 Documentation Update
- [ ] 🔖 Release
- [X] 🚩 Other

## Description
Updated the PR template.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
image

## Added to documentation?

- [ ] 📜 readme
- [X] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

### CODE REVIEW

**Review by Programing-School — APPROVED** [review]
Nice One Guy

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -31,16 +30,10 @@
 
 
 
-## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
```
Comment (mudit023) [inline .github/PULL_REQUEST_TEMPLATE.md:34]: change it to:
## Mobile & Desktop Screenshots or GIFs(If Template have animations)

**Review by mudit023 — COMMENTED** [review]
Look into the requested change

**Comment by Programing-School** [comment]
@mudit023 Can I tell you a new idea here in this pull request ?

**Comment by Programing-School** [comment]
> @mudit023 Can I tell you a new idea here in this pull request ?

Make A landing Page for our templates(I will make it using react js)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (pranavgoel29) [reply .github/PULL_REQUEST_TEMPLATE.md:34]: This is removed one

**Review by mudit023 — COMMENTED** [review]
Done

**Comment by mudit023** [comment]
> > @mudit023 Can I tell you a new idea here in this pull request ?
> 
> Make A landing Page for our templates(I will make it using react js)

@Programing-School yeah do it.

**Comment by mudit023** [comment]
Let's discuss it on slack

**Pull request merged**